pep-9999.rst

```diff
@@ -10,12 +10,14 @@ Created:
 
 Abstract
 ========
-
-This PEP describes the Python Packaging Governance for all tools 
-that are included as part of the PyPA. The Packaging Council will 
-govern all PyPA tools, represent the community that supports these 
-tools and act as a representative of PyPA tools. From here on, this 
-Packaging council will be called the Python Packaging Authority (PyPA).
+This PEP describes the Python Packaging Governance for 
+all tools that are included as part of the PyPA. The Packaging 
+Council will govern all PyPA tools, represent the community 
+that supports these tools and act as a representative of PyPA 
+tools. From here on, this Packaging council will be called the 
+Python Packaging Authority (PyPA). This PEP will supersede 
+[PEP 609](https://peps.python.org/pep-0609/) 
```

Comment:
We can use this [Sphinx role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-pep) as a shortcut for PEP links:
```suggestion
:pep:`609`
```